### PR TITLE
Sync CircleCI Orb version

### PR DESCRIPTION
**Pull request by SuperQ**
Add sync of the Prometheus CircleCI Orb version to the sync script.
* Add `yq` to the repo sync CI job.

NOTE: This will re-format the down-stream yaml config, it does not
preserve whitespace and enforces some indenting rules. It does preserve
comments.

Signed-off-by: Ben Kochie <superq@gmail.com>

### CODE REVIEW

**Comment by roidelapluie** [comment]
I appreciate. We should also sync promu version because the last orb requires the last promu.

**Comment by SuperQ** [comment]
The promu version is managed already in the Makefile.common sync.

File: .circleci/config.yml [hunk 1]
```diff
@@ -110,6 +110,7 @@ jobs:
     executor: golang
     steps:
     - checkout
+    - run: curl -sL https://github.com/mikefarah/yq/releases/download/v4.6.3/yq_linux_amd64 -O "${PATH%%:*}/yq" && chmod -v +x "${PATH%%:*}/yq"
```
Comment (roidelapluie) [inline .circleci/config.yml:113]: can we hardcode the checksum and verify it? this script has access to our secret

Comment (SuperQ) [reply .circleci/config.yml:113]: Done.

**Review by roidelapluie — APPROVED** [review]

**Pull request merged**